tests/client/core/api/test_api.js

```diff
@@ -142,6 +142,53 @@ describe('api', () => {
     });
   });
 
+  describe('versionDetail api', () => {
+    function mockResponse() {
```

Comment:
This doesn't look like it needs to be a function. You could just reference the resolved promise object.